creation/probes.md

```diff
@@ -169,3 +169,30 @@ filled:
 Note that in the case of Readiness Probes, it is also possible to set a
 `failureThreshold` and a `successThreashold` to only flip between ready
 and non-ready state if the probe succeeded or failed multiple times.
+
+Dual-stack considerations
+-------------------------
+
+Some context is needed to understand the limitations imposed by a dual-stack
+network configuration on readiness - or liveness - probes. Users must be
+fully aware that a dual-stack configuration is currently only available when
+using a masquerade binding type. Furthermore, it must be recalled that
+accessing a VM using masquerade binding type is performed via the pod IP
+address; in dual-stack mode, both IPv4 and IPv6 addresses can be used to reach
+the VM.
+
+Dual-stack networking configurations have a limitation when using HTTP / TCP
+probes - you **cannot probe the VMI by its IPv6 address**. The reason for this
+is the
+[HTTP](https://kubernetes.io/docs/reference/generated/kubernetes-api/v1.18/#httpgetaction-v1-core)
+(or
+[TCP](https://kubernetes.io/docs/reference/generated/kubernetes-api/v1.18/#tcpsocketaction-v1-core))
+probe actions `host` optional field default to the pod's IP address - which, is
+currently **always** the IPv4 address.
+
+Since the probe must be specified when defining the VMI, it is impossible to
+fill the `host` field with the IP address - we do not have the pod IP address
+yet.
+
+A work-around for this would be to expose the VMI server replying to the probe
```

Comment:
I think this last line would benefit by an example, or a link to somewhere it is done like so.

Reply:
I get the feeling that's a bit besides the point ... What I could do is refer the users to https://github.com/kubevirt/user-guide/blob/master/usage/network-service-integration.md#expose-virtualmachineinstance-as-a-clusterip-service , which explains how to do that . 

It would probably be safer to just remove the whole paragraph, since I have *never* tried it (I just don't see why it wouldn't work) ...

I'll make sure it works, then either refer the users to the service API, *or* get rid of this.

Reply:
Despite having the DNS resolution correctly evaluated, I couldn't get a liveness probe to work using the work-around.

As such, since I don't have time to debug the scenario, I prefer to just remove the section about the work-around.